openapi/nexus.json

```diff
@@ -19798,7 +19798,8 @@
         "type": "string",
         "enum": [
           "count",
-          "bytes"
+          "bytes",
+          "seconds"
```

Comment:
This needs to be integrated into the CRDB SQL introduced in #6005. If we land this after, that would mean a new migration to add that enum variant. If we land this before, make sure we add that variant to the SQL.